app/cdap/components/ParsingDrawer/styles.ts

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { makeStyles } from '@material-ui/styles';
+
+export const useStyles = makeStyles(() => {
+  return {
+    parsingDrawerStyles: {
+      width: 460,
+      height: '100%',
+      padding: '5px 20px 5px 30px',
+    },
+    panelStyles: {
+      height: '100%',
+    },
+    paper: {
+      top: '46px',
+    },
+    headerStyles: {
+      height: '60px',
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
+    headingStyles: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    pointerStyles: {
+      cursor: 'pointer',
+    },
+    headingTextStyles: {
+      fontFamily: 'Noto Sans',
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: 20,
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#000000',
+    },
+    headerRightStyles: {
+      display: 'flex',
+      alignItems: 'center',
+    },
+    importIconStyles: {
+      marginRight: '10px',
+    },
+    importSchemaTextStyles: {
+      fontFamily: 'Noto Sans',
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: 14,
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#000000',
+    },
+    dividerLineStyles: {
+      width: 1,
+      height: 28,
+      backgroundColor: '#DADCE0',
+      margin: '0 15px',
+    },
+    checkboxStyles: {
+      display: 'flex',
+      width: '100%',
+      marginBottom: 0,
+    },
+    labelTextStyles: {
+      fontFamily: 'Noto Sans',
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: 14,
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#5F6368',
```

Comment:
use number directly instead of using number with px everywhere
and use MUI colors if possible

Reply:
Done and Mui Color not available